command_test.go

```diff
@@ -5,35 +5,33 @@ import (
 	"path/filepath"
 	"testing"
 
-	mesos "github.com/mesos/mesos-go/api/v1/lib"
+	"github.com/mesos/mesos-go/api/v1/lib"
 	"github.com/stretchr/testify/assert"
 )
 
-func TestIfNewCancellableCommandReturnsCommand(t *testing.T) {
-	os.Environ()
-	commandInfo := newCommandInfo("./sleep 100", "ignored", false, []string{"ignored"}, map[string]string{"one": "1", "two": "2"})
+func TestIfNewCancellableCommandReturnsCommandWithoutExecutorEnv(t *testing.T) {
+	os.Setenv("ALLEGRO_EXECUTOR_TEST_1", "x")
```

Comment:
Add defer with `os.Unsetenv` for these variables, you are modifying env for all tests here.

Reply:
Fixed